charts/openebs/Chart.yaml

```diff
@@ -24,19 +24,19 @@ maintainers:
     email: shovan.maity@mayadata.io
 dependencies:
   - name: openebs-ndm
-    version: "1.6.1"
+    version: "1.7.1"
     repository: "https://openebs.github.io/node-disk-manager"
     condition: openebs-ndm.enabled
   - name: localpv-provisioner
-    version: "2.12.1"
+    version: "3.0.1"
     repository: "https://openebs.github.io/dynamic-localpv-provisioner"
     condition: localpv-provisioner.enabled
   - name: cstor
-    version: "2.12.1"
+    version: "3.0.1"
     repository: "https://openebs.github.io/cstor-operators"
     condition: cstor.enabled
   - name: jiva
-    version: "2.12.2"
+    version: "3.0.0"
     repository: "https://openebs.github.io/jiva-operator"
     condition: jiva.enabled
   - name: zfs-localpv
```

Comment:
Let's bump LVM from `0.8.4` to [0.8.5](https://github.com/openebs/lvm-localpv/blob/876b40f0da2d36ebff77fd95b8a33062db1d2ee7/deploy/helm/charts/Chart.yaml#L4)

Reply:
Jiva at 3.0.1